etc/tutorial/cylc-forecasting-suite/bin/get-observations

```diff
@@ -1,10 +1,24 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
-# Copyright (C) 2012-2020 British Crown (Met Office) & Contributors - GNU V3+.
-# This is illustrative code developed for tutorial purposes, it is not
-# intended for scientific use and is not guarantied to be accurate or correct.
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Doesn't the Rose licence differ from Cylc?

Reply:
Yep, althoug I think that this is superficial. Will fix.

Reply:
I think I've fixed this, although I notice that a number of other copyright notices have leaked from Cylc into Rose they are not related to this change, and I'd prefer to address them separately.